pkg/operator/ceph/cluster/nodedaemon/exporter.go

```diff
@@ -153,16 +157,21 @@ func getCephExporterChownInitContainer(cephCluster cephv1.CephCluster) corev1.Co
 func getCephExporterDaemonContainer(cephCluster cephv1.CephCluster, cephVersion cephver.CephVersion) corev1.Container {
 	cephImage := cephCluster.Spec.CephVersion.Image
 	dataPathMap := config.NewDatalessDaemonDataPathMap(cephCluster.GetNamespace(), cephCluster.Spec.DataDirHostPath)
-	configDir := path.Join(cephCluster.Spec.DataDirHostPath, cephCluster.Namespace)
-	cephConfFile := path.Join(configDir, fmt.Sprintf("%s.config", cephCluster.Namespace))
-	volumeMounts := append(controller.DaemonVolumeMounts(dataPathMap, "", cephCluster.Spec.DataDirHostPath), v1.VolumeMount{Name: "ceph-conf-dir", MountPath: configDir})
+	volumeMounts := controller.DaemonVolumeMounts(dataPathMap, "", cephCluster.Spec.DataDirHostPath)
+	// FIX: Use an exporter keyring instead of the admin keyring
```

Comment:
Additional future work: all ceph daemons (except NFS) mount the `rook-ceph-config-override` configmap to `/etc/ceph/ceph.conf` so that users can override configs via configmap. Does the exporter have config options that can be set by configmap? If so, we should make sure that's added to the pods as well.

Reply:
Actually it looks like the exporter has that configmap mounted, so we should be covered. I see it in my test cluster on the exporter pod spec:
```
    Mounts:
      /etc/ceph from rook-config-override (ro)
```